### PR TITLE
#174566483 Set up the sequelize enviroment

### PR DESCRIPTION
**Pull request by donaldcrane**
### What does this PR do?

This PR sets up the sequelize environment by adding sequelize and postgres packages.

### Description of tasks to be completed?

-  Install sequelize package 
-  Install postgres package 
-  install sequelize-cli package 
   
### How should this be manually tested/checked?

-   Clone the PR
-   Cd into the branch `cd ch-setup-seq-env-174566483`
-   Install the dependencies using npm install

    
### What are the relevant pivotal tracker stories?
[#174566483](https://www.pivotaltracker.com/n/projects/2463429/stories/174566483)

### CODE REVIEW

**Review by Fiyin-Anne — COMMENTED** [review]
I followed the steps but `sequelize init` returns an error message saying `sequelize : The term 'sequelize' is not recognized as the name of a cmdlet, function, script file, or operable program`.

**Comment by donaldcrane** [comment]
sequelize-cli should be installed globally, then run sequelize init (this should create the working enviroment files including the config file)

**Comment by Fiyin-Anne** [comment]
> sequelize-cli should be installed globally, then run sequelize init (this should create the working enviroment files including the config file)

Please add this to your description.

**Review by bellogo — COMMENTED** [review]
After `npm install`
i ran: `sequelize init` and i got this

ERROR: The file config\config.json already exists. Run command with --force to overwrite it.

**Comment by AbonyiXavier** [comment]
You are supposed to  create a .sequelizerc file in the root directory of your project, change config.json to config.js to enable us make use of our environment variables for database connection.

**Comment by donaldcrane** [comment]
> 
> 
> After `npm install`
> i ran: `sequelize init` and i got this
> 
> ERROR: The file config\config.json already exists. Run command with --force to overwrite it.
> 
> Is this the expected behavior?

you are not expected to overwrite it  because i already created it, you could just modify the codes inside(so u could skip `sequelize init`

**Comment by donaldcrane** [comment]
> 
> 
> You are supposed to create a .sequelizerc file in the root directory of the project, change config.json to config.js to enable us make use of our environment variables for the database connection.

i have done that

**Review by Fiyin-Anne — APPROVED** [review]

**Review by bellogo — APPROVED** [review]

**Comment by AbonyiXavier** [comment]
There is one more package left to install, which is  sequelize-cli , this package will enable us to generate our models, migrations and seeders folders using this command npx sequelize-cli init.

File: src/models/index.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+'use strict';
```
[the remark this answers is not in the document]
Comment (codeliezel) [reply src/models/index.js:1]: lint all your codes using augusta command ad=nd re-commit

**Review by codeliezel — CHANGES_REQUESTED** [review]
@donaldcrane  the configuration you did in the config/config.js is the wrong location, do it in the database/config file.

good job

**Comment by codeliezel** [comment]
@donaldcrane you have not even done what I asked you to do, copy all the codes away from the config/config.js and add it to database/config.js

**Comment by codeliezel** [comment]
@donaldcrane also in your pr description, npm install is enough, that will install all dependencies.. update your pr

**Review by bellogo — APPROVED** [review]

**Review by Fiyin-Anne — APPROVED** [review]

**Review by Augusta-E — APPROVED** [review]

**Review by codeliezel — APPROVED** [review]
Good job

**Pull request merged**